atomic_reactor/util.py

```diff
@@ -649,18 +649,19 @@ def default(self):
         return self.v2 or self.v1
 
 
-def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
-                         versions=('v1', 'v2')):
+def query_registry(image, registry, digest=None, insecure=False, dockercfg_path=None,
```

Comment:
What if we just query the registry once? We could enhance `ManifestDigest` class to also store config contents from manifest.